lib/misc/idLinux.c: add uClibc support

uClibc does not have a secure_getenv function, so we use the
Android method.

Signed-off-by: Karoly Kasza <kaszak@gmail.com>

--- openvmtools-stable-9.10.0.orig/open-vm-tools/lib/misc/idLinux.c	2015-06-17 10:04:00.000000000 +0200
+++ openvmtools-stable-9.10.0/open-vm-tools/lib/misc/idLinux.c		2015-06-17 10:04:00.000000000 +0200
@@ -997,7 +997,7 @@
 static Bool
 IdIsSetUGid(void)
 {
-#if defined(__ANDROID__)
+#if defined(__ANDROID__) || defined(__UCLIBC__)
    /* Android does not have a secure_getenv, so be conservative. */
    return TRUE;
 #else
